stoservice/src/test/java/com/tutrit/stoservice/repository/UserRepositoryTest.java

```diff
@@ -0,0 +1,47 @@
+package com.tutrit.stoservice.repository;
+
+import com.tutrit.stoservice.bean.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UserRepositoryTest {
+
+    UserRepository userRepository;
+
+    @BeforeEach
+    void SetUp() {
```

Comment:
method name should started from lower case

Reply:
oops)